content/post/invitation-to-test.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Invitation to test : IR hosting service"
+date: 2020-09-28T17:00:00+02:00
+draft: false
+tags: ["iraas", "repository", "shared portal", "dedicated portal"]
+---
+
+One of the goals of the [SONAR project](https://sonar.ch/) is to set up a hosting service for institutional repositories (IRaaS - Institutional Repository as a Service). [Two variants](https://sonar.ch/iraas/) of this service are proposed: a shared portal and dedicated portals. In anticipation of its first production launch, this service is currently providing a public test instance at the address [sonar.test.rero.ch](https://sonar.test.rero.ch).
+
+<!--more-->
+
+In order to respond to the interest that it has been attracting in our community, the SONAR team is inviting all interested institutions to test this service. Just send a short message to [info.sonar@rero.ch](mailto:info.sonar@rero.ch) with the **name of the institution** and the **name and email address of the person in charge** (who will be assigned the [Admin role](https://sonar.test.rero.ch/help/user_roles/)).
+
+
+<img class="image fit" src="/images/invitation-to-test.png" alt="SONAR public test portal" title="SONAR public test portal" >
+
+
+The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles and the document deposit and validation process. Some features are still under development, and will progressively become visible. The development backlog is available (live) on [GitHub](https://github.com/orgs/rero/projects/4).
```

Comment:
The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles and the document deposit and validation process.
to
The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles, as well as document deposit and validation processes.

Reply:
It's a single process involving deposit and validation.